cli/src/main/java/com/devonfw/tools/ide/tool/squirrelsql/SquirrelSqlUrlUpdater.java

```diff
@@ -0,0 +1,44 @@
+package com.devonfw.tools.ide.tool.squirrelsql;
+
+import com.devonfw.tools.ide.url.model.folder.UrlVersion;
+import com.devonfw.tools.ide.url.updater.GithubUrlUpdater;
+import com.devonfw.tools.ide.version.VersionIdentifier;
+
+/**
+ * {@link GithubUrlUpdater} for SquirrelSQL
+ */
+public class SquirrelSqlUrlUpdater extends GithubUrlUpdater {
+
+  public static final VersionIdentifier MIN_SQUIRRELSQL_VID = VersionIdentifier.of("4.4.0");
+
+  @Override
+  protected String getTool() {
+
+    return "squirrelsql";
+  }
+
+  @Override
+  protected String getGithubOrganization() {
+
+    return "squirrel-sql-client";
+  }
+
+  @Override
+  protected String getGithubRepository() {
+
+    return "squirrel-sql-code";
+  }
+
+  @Override
+  protected void addVersion(UrlVersion urlVersion) {
+
+    VersionIdentifier vid = urlVersion.getVersionIdentifier();
+    if (vid.compareVersion(MIN_SQUIRRELSQL_VID).isGreater()) {
+      String baseUrl = "https://squirrel-sql.sourceforge.io/#installation";
```

Comment:
This is not a download link that can work. Anchors are only processed by the client (browser) and can be handled by JavaScript. IMHO anchor links are not even transferred to the URL in the request send to the server.
Am I wrong with this? Or was this code never really tested?